libbeat/docs/processors-using.asciidoc

```diff
@@ -531,6 +532,52 @@ section.
 NOTE: If you define an empty list of fields under `include_fields`, then only
 the required fields, `@timestamp` and `type`, are exported.
 
+[[rename-fields]]
+=== Rename fields from events
+
+The `rename` processor specifies a list of fields that should be renamed. The
+list of fields to be renamed are listed under `fields`. Each entry contains a
+`from: old-key` and a `to: new-key` pair. `from` is the origin and `to` the
+target name of the field.
+
+The renaming of fields can be useful in case two fields potentially conflict.
+
+Renaming fields can be useful in cases where field names cause conflicts. For
+example if an event has two fields, `a` and `a.b`, that are both assigned a
+scalar values (e.g. `{"a": 1, "a.b": 2}`) this will result in an error at ingest
+time from Elasticsearch. This is because the value of a cannot simultaneously be
+a scalar and an object. To prevent this rename_fields can be used to rename a to
```

Comment:
same comment here. Use back ticks to wrap field names, especially if they can be confused with articles.